packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -423,6 +421,7 @@ async function waitForPageIdle({ page, waitForPageIdleMillis, maxWaitForPageIdle
         timeout = activityHandler(); // We call this once manually in case there would be no requests at all.
         page.on('request', activityHandler);
         page.on('framenavigated', activityHandler);
+        // @ts-expect-error TODO: Argument of type '"targetcreated"' is not assignable to parameter of type 'keyof PageEventObject'.
         page.on('targetcreated', newTabTracker);
```

Comment:
you could type cast, not great but if its some nonstandard event, better to do that instead ts-expect-error 

```suggestion
        page.on('targetcreated' as keyof PageEventObject, newTabTracker);
```